src/main/java/com/alex/eshop/service/CategoryService.java

```diff
@@ -26,6 +27,18 @@ public List<CategoryDTO> getAllCategories() {
     }
 
     public CategoryDTO getCategory(Long id) {
-        return categoryMapper.toDto(categoryRepository.findById(id).orElseThrow(()->new DataNotFound("There is no category with id " + id)));
+        return categoryMapper.toDto(categoryRepository.findById(id)
+                .orElseThrow(()->new DataNotFound("There is no category with id " + id)));
+    }
+    public void createCategory(CategoryDTO categoryCreateDTO){
+       categoryRepository.save(categoryMapper.toEntity(categoryCreateDTO));
+    }
+    public void updateCategory(CategoryDTO categoryUpdateDTO){
+        categoryRepository.existsById(categoryUpdateDTO.getId());
```

Comment:
What will happen if you try to update category, that does not exists in database?